app/main.py

```diff
@@ -1,7 +1,31 @@
-def get_weather() -> None:
-    # write your code here
-    pass
+import os
+
+import requests
+import json
+
+URL = "http://api.weatherapi.com/v1/current.json?"
+
+
+def get_weather(query_params: dict) -> None:
+    query_params["key"] = os.environ["API_KEY"]
+    params = "&".join([f"{k}={v}" for k, v in query_params.items()])
+
+    print("Performing request to Weather API...")
+    request = requests.get(URL + params)
```

Comment:
![image](https://user-images.githubusercontent.com/44442087/222123225-4e909a7d-2c7b-46a9-a42d-fc5e11f55e91.png)
I meant to use params argument, not to append it to URL
pass params as dict